.github/workflows/shopify-cli.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Install dependencies
         run: yarn install --ignore-engines
       - name: Build the fixture app
-        run: yarn shopify app build --path ./fixtures/app --skip-dependencies-installation
+        run: yarn shopify app build --path ./fixtures/app
```

Comment:
The fixtures are part of the workspace in this repository and therefore we don't need to install dependencies at runtime. I've created an empty `yarn.lock` file in the fixture's directory for the CLI to detect this project as a Yarn project. That results to a noop `yarn install` operation.